docs/development.md

```diff
@@ -35,12 +37,22 @@ export const invocationConfig: IntegrationInvocationConfig = {
     const { config } = context.instance;
 
     if (!config.clientId || !config.clientSecret) {
-      throw new IntegrationInstanceConfigError(
+      throw new IntegrationValidationError(
         'Config requires a clientId and clientSecret to be provided',
       );
     }
 
-    await validate(config);
+    const apiClient = createAPIClient(config);
```

Comment:
I think you're right, this isn't sufficient as is, should probably even have a `createAPIClient.ts` file so that the project compiles after they create it.